css/values.css

```diff
@@ -0,0 +1,544 @@
+.empowering h2 {
+	font-size: var(--font28);
+	font-weight: var(--font-weight-bold);
+	line-height: var(--line-height32);
+	color: var(--color-onyx);
+	margin-bottom: 20px;
+}
+.empowering p {
+	text-align: left;
+	color: var(--color-onyx);
+	padding: 0px var(--spacing20);
+	margin-bottom: 60px;
+}
+/* Adds Red Block behind this img - this changes in desktop version somehow */
+.empowering-img-block {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	background-color: var(--color-amaranth);
+}
+.empowering-img-block img {
+	margin: 30px;
+}
+
+/* ===================================
+Section Styles (background, padding, etc.)
+===================================
+*/
+
+/* 
+===================================
+Media Queries (overriding styles for different screen sizes)
+
+Note: Here we are using a mobile-first approach, so we start with the smallest screen size and work our way up.
+Start by writing styles for the mobile version of the design on the lines above.
+Anything that needs to change for larger screens should be added to the media queries below.
+
+Feel free to change the min/max-width values to whatever you need. Use variables.
+===================================
+*/
+
+/* Tablet and Larger */
+@media screen and (min-width: 768px) {
+	/* Add styles that apply to table and desktop */
+}
+
+/* Tablet Only */
+@media screen and (min-width: 768px) and (max-width: 1024px) {
+	/* Add styles that only apply to tablet. These are usually fixes to get responsiveness right */
+}
+
+/* Desktop Only */
+@media screen and (min-width: 1024px) {
+	/* Add styles that apply specifically to desktop */
+
+	/* adjusting font sizes and styling for p tags */
+
+	/* Values header and p needs to be on top of the hero */
+	.desktop-parent {
+		display: flex;
+		flex-direction: column;
+		gap: 20px;
+		margin: 190px auto 193px auto;
+		max-width: 1328px;
+	}
+	.values-main {
+		display: flex;
+		position: absolute;
+		left: 50%;
+		top: 45%;
+	}
+	.values-card {
+		margin: 0px 10px;
+	}
+	.values-card p {
+		margin: var(--spacing20);
+		font-size: var(--font21);
+		font-weight: var(--font-weight-regular);
+		line-height: var(--line-height28);
+	}
+	.copperback,
+	.authenticity,
+	.education {
+		margin-right: 10px;
+	}
+	.sustainability-text,
+	.grapeback,
+	.technology {
+		margin-left: 10px;
+	}
+	.values-card h2 {
+		margin: 20px 0px 0px 20px;
+		font-size: var(--font36);
+		font-weight: var(--font-weight-bold);
+		line-height: var(--line-height40);
+		text-align: left;
+	}
+	.values-head p {
+		text-align: left;
+		margin: 0;
+		font-size: var(--font21);
+		font-weight: var(--font-weight-regular);
+		line-height: var(--line-height26);
+	}
+
+	.onyxback,
+	.grapeback,
+	.tealback {
+		padding-right: 44px;
+	}
+
+	/* Cards need to change from column to row with some stylized sizing and padding*/
+	.desktop-group {
+		display: flex;
+		flex-direction: row;
+		height: 290px;
+	}
+	/* Gradients */
+	.hero {
+		width: 100vw;
```

Comment:
This is causing some slight horizontal scrolling on desktop for me. I think you can delete this because `<section>` defaults to 100% width. 

Fyi: This is true for divs and other block-level elements, like div, header, ul, etc.

<img width="536" alt="Screen Shot 2024-01-22 at 11 04 33 AM" src="https://github.com/indigenousfriends/itlct-website-redesign/assets/704697/c8bbee9a-f2f3-439b-a58e-ac9f2db870f6">

Reply:
removed - had to add a little margin to the text portions to scoot them away from the left edge